contrib/test_jenkins.sh

```diff
@@ -1014,6 +1015,63 @@ test_memtrack() {
 	UCX_MEMTRACK_DEST=stdout UCX_HANDLE_ERRORS=none UCX_MEMTRACK_LIMIT=412MB ./test/apps/test_memtrack_limit |& grep -C 100 'reached'
 }
 
+test_namespace() {
+	# Make sure to try to use CMA when possible
+	# Expect fallback on SYSV
+	args="-t ucp_get -s 9999999 -n 5"
+
+	perftest="$ucx_inst/bin/ucx_perftest"
+	info="./src/tools/info/ucx_info"
+
+	echo "==== Running perftest namespace positive tests ===="
+
+	export UCX_LOG_LEVEL="debug"
+
+	echo "==== Running perftest default and non-default USER namespace test for posix ===="
+	export UCX_TLS="$tl,sysv"
+	cmd="$perftest $args -p $server_port"
+	step_server_port
+	$cmd &
```

Comment:
why need to use the local var cmd here?

Reply:
trying to reuse since we use the same cli two time